products/alibaba-ack.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Alibaba ACK
+category: service
+tags: alibaba managed-kubernetes
+iconSlug: alibabacloud
+permalink: /alibaba-ack
+alternate_urls:
+-   /ack
+-   /alibaba-container-service-for-kubernetes
+versionCommand: kubectl version
+releasePolicyLink: https://www.alibabacloud.com/help/en/ack/ack-managed-and-ack-dedicated/user-guide/support-for-kubernetes-versions
+releaseDateColumn: true
+eolColumn: End of Support
+
+releases:
+-   releaseCycle: "1.32"
+    releaseDate: 2025-01-01
+    eol: 2026-01-31
+    latest: '1.32.1-aliyun.1'
```

Comment:
```suggestion
```

This will be hard to maintain without automation, and automation is very hard on the CI since each release page must be scraped to get the latest version. Also we prefer also documenting `latestReleaseDate` when `latest` is set.

Is it possible to drop `latest` (and set `releaseColumn: false`) ?